### PR TITLE
GEOMESA-864 Kafka Data Store Cache Cleanup

### PR DESCRIPTION
**Pull request by Afterglow375**
* Implemented thread to clean up guava cache every second

Signed-off-by: Alex Tatusko <atatusko@ccri.com>

### CODE REVIEW

File: geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/LiveKafkaConsumerFeatureSource.scala [hunk 1]
```diff
@@ -100,6 +100,12 @@ class LiveKafkaConsumerFeatureSource(entry: ContentEntry,
     }
   })
 
+  val ses = new ScheduledThreadPoolExecutor(1)
```
Comment (elahrvivaz) [inline geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/LiveKafkaConsumerFeatureSource.scala]: should add this to the shutdown hook on line 46

**Comment by elahrvivaz** [comment]
looks good.

File: geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/LiveKafkaConsumerFeatureSource.scala (hunk 1 above)
Comment (elahrvivaz) [inline geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/LiveKafkaConsumerFeatureSource.scala]: This should be added to the shutdown hook on line 46

File: geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/LiveKafkaConsumerFeatureSource.scala [hunk 2]
```diff
@@ -100,6 +100,12 @@ class LiveKafkaConsumerFeatureSource(entry: ContentEntry,
...
+
+  ses.scheduleAtFixedRate(new Runnable() {
```
Comment (elahrvivaz) [inline geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/LiveKafkaConsumerFeatureSource.scala]: do we need to run this if the cache is not set to auto-expire entries based on timeout? That is configurable via the expirationPeriod option.

**Comment by anthonyccri** [comment]
622a735..18c64ef

**Pull request closed without merging**